move.c

```diff
@@ -20,7 +20,7 @@ coord2bstr(char *buf, coord_t c, struct board *board)
 		return "resign";
 	} else {
 		/* Some GTP servers are broken and won't grok lowercase coords */
-		snprintf(buf, 4, "%c%d", toupper(asdf[coord_x(c, board) - 1]), coord_y(c, board));
+		snprintf(buf, 4, "%c%u", toupper(asdf[coord_x(c, board) - 1]), coord_y(c, board) % 100);
```

Comment:
Not sure i understand why we need % 100 now, ok to remove it ?

Reply:
GCC isn't sure of the integer's range. It could be negative and that adds one more character despite modulo arithmetic.